spec/api/datastore/v2/applications_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Datastore::V2::Applications do
+  let(:valid_payload) do
+    file_fixture('crime_applications/696dd4fd-b619-4637-ab42-a5f4565bcf4a.json').read
```

Comment:
You can use a valid fixture from the schemas gem, for example:

`LaaCrimeSchemas.fixture(1.0).read`

This way we don't have to copy&paste fixtures across the different repos and is easier to keep them updated or add new ones if needed 👍

Reply:
Nice! Thank you!